core/models.py

```diff
@@ -162,10 +138,27 @@ def __str__(self: "PostVote") -> str:
 
     def save(self: "PostVote", *args: int, **kwargs: int) -> None:
         super().save(*args, **kwargs)
-        post_vote = PostVote.objects.filter(post=self.post)
-        self.post.up_votes = post_vote.filter(choice=PostVote.UPVOTE).count()
-        self.post.down_votes = post_vote.filter(choice=PostVote.DOWNVOTE).count()
-        self.post.skip_version_check = True
-        self.post.save()
-        self.post.skip_version_check = False
+        post_votes = PostVote.objects.filter(post=self.post)
+        up_votes = post_votes.filter(choice=PostVote.UPVOTE).count()
+        down_votes = post_votes.filter(choice=PostVote.DOWNVOTE).count()
+        Post.objects.filter(pk=self.post.pk).update(up_votes=up_votes, down_votes=down_votes)
+
+
+class Image(models.Model):
+    post = models.ForeignKey(Post, on_delete=models.CASCADE, related_name="images")
+    image = models.ImageField()
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    order = models.PositiveIntegerField(default=0)

```

Comment:
default 9999999 was defined on purpose. Now, every image will have zero. Imagine you have 5 images, sorted, 0 to 4. Now you add one more. And it has 0... where it will go?
if it has 9999999 it will be last.

Reply:
Understood and agreed. Thank you.